app/MMB/src/appRegistry/ManageMyBookingPackage.js

```diff
@@ -11,5 +12,9 @@ type Props = {|
 |};
 
 export default class ManageMyBookingPackage extends React.Component<Props> {
-  render = () => <NavigationStack screenProps={this.props} />;
+  render = () => (
+    <MMBContext.Provider>
+      <NavigationStack screenProps={this.props} />
+    </MMBContext.Provider>
```

Comment:
This is little bit odd. You should set `accessToken` here directly and completely get rid of `setAccessToken` action. We wanted to avoid prop-drilling but this is not very convincing...